external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/DockerKrbJDBCIntegrationSuite.scala

```diff
@@ -100,7 +100,7 @@ abstract class DockerKrbJDBCIntegrationSuite extends DockerJDBCIntegrationSuite
   }
 
   override def dataPreparation(conn: Connection): Unit = {
-    conn.prepareStatement("CREATE TABLE bar (c0 text)").executeUpdate()
+    conn.prepareStatement("CREATE TABLE bar (c0 VARCHAR(8))").executeUpdate()
```

Comment:
DB2 doesn't support text.

Reply:
I'd be surprised if this change affects others, but it may be worth to test others manually and mention the result.

Reply:
When doing modifications I'm always re-executing all of them. This has happened here too.

Reply:
And not to avoid the question all passed :)